### ISSUE TITLE
go get aws-sdk-go broken with v1.2.1 (2021-03-12) release

### ISSUE THREAD

**Issue by jconning** [aws/aws-sdk-go-v2#1180]
We have a daily build script that executes "go get aws-sdk-go" on a long lived server that runs everyday and has been running successfully without changes for well over a year. It started failing on March 13, 2021. In the logs was an error message about master branch being renamed to main branch.

Our engineers were not familiar with how "go get" works under the hood and were befuddled for days as to why it stopped working. I saw a blog post about people renaming master branch due to social inclusivity reasons. I theorized that "go get" was actually updating a local git clone and that if we deleted the local clone then it would start working again. Sure enough that was it.

The v1.2.1 (2021-03-12) release notes make no mention of the branch name but the December 23, 2021 release notes do. Our daily build broke immediately after the March 12 release. Therefore I think you should modify the release notes because the impact of the branch name change was felt in March, not December.

**Comment by skmcgrail** [comment]
Just to alleviate confusion here, are you referring to the AWS SDK for Go (V1) Product which changed the HEAD tracking branch as part of [`v1.37.31`](https://github.com/aws/aws-sdk-go/releases/tag/v1.37.31)? We have since added an announcement to that release to call out this specific behavior when using `GOPATH`. I think this issue was cut to the incorrect product repository, but just want to verify you aren't seeing another issue with the V2 version of the SDK.

**Comment by jconning** [comment]
Sean:
Yes you are right. I filed a bug against the wrong repo. I wasn't aware that there was a v2 repo.
We didn't change our GOPATH but what we did do is remove the local git clone (that go get created) on the long-lived host. That fixed it.
ThanksJim
    On Thursday, March 18, 2021, 11:30:10 AM PDT, Sean McGrail ***@***.***> wrote:  
 
 


Just to alleviate confusion here, are you referring to the AWS SDK for Go (V1) Product which changed the HEAD tracking branch as part of v1.37.31? We have since added an announcement to that release to call out this specific behavior when using GOPATH. I think this issue was cut to the incorrect product repository, but just want to verify you aren't seeing another issue with the V2 version of the SDK.
